docroot/themes/humsci/humsci_basic/src/scss/_main.scss

```diff
@@ -87,7 +87,7 @@
   // 6. Components
   //    Specific UI components. This is where majority of our work takes place
   //    and our UI components are often composed of Objects and Components
-  'components/pattern.horizontal-card',
+  'components/card',
```

Comment:
Love this change!

Reply:
Yeah, we don't need to prefix our scss partials. We will do that sometimes if the code base is small and we don't nest the partials in directories but since these are nested we don't need the prefixes.